src/middlewares/validations/auth.validation.js

```diff
@@ -0,0 +1,19 @@
+import Joi from '@hapi/joi';
+
+export default (req, res, next) => {
```

Comment:
Change this to a named export since this file will contain multiple validations schema